groups/models.py

```diff
@@ -161,6 +161,20 @@ def delete_state(self):
     def is_deleted(self):
         return self.state == self.STATE_DELETED
 
+    def subscribers(self):
```

Comment:
Any suggestions? It's not the easiest thing to name - users who are interested in this comment's discussion or group and didn't post this comment.